app/src/main/java/org/oppia/android/app/testing/NavigationDrawerTestActivity.kt

```diff
@@ -76,7 +81,7 @@ class NavigationDrawerTestActivity :
     val recentlyPlayedActivityParams =
       RecentlyPlayedActivityParams
         .newBuilder()
-        .setProfileId(ProfileId.newBuilder().setInternalId(internalProfileId).build())
+        .setProfileId(ProfileId.newBuilder().setInternalId(profileId.internalId).build())
```

Comment:
Couldn't we just use ``profileId`` itself, here? The copy doesn't seem necessary.

Reply:
Updated makes sense.